rubocop.yml

```diff
@@ -1,4 +1,48 @@
 # LTV Rubocop config
+#
+# This is a living config; if a rule doesn't make sense, configure it or
+# disable it to align with our style guidelines.
+#
+# Documentation on specific cops can be found in the Rubocop docs:
+#
+#   https://docs.rubocop.org/rubocop/index.html
 
+inherit_mode:
+  merge:
+    - Exclude
+AllCops:
+  NewCops: disable
+  Exclude:
+    - 'bin/{rails,rake,resque,rspec,spring}'
+    - 'Gemfile'
+    - 'Rakefile'
+    - 'config/unicorn.rb'
 Layout/LineLength:
   Max: 100
+Lint/AmbiguousBlockAssociation:
+  Exclude:
+    - "spec/**/*"
+Metrics/BlockLength:
+  Exclude:
+    - 'Rakefile'
+    - '**/*.rake'
+    - 'spec/**/*.rb'
+Style/BlockComments:
+  Exclude:
+    - 'db/migrations/*'
```

Comment:
This rule is specific to the Core practice where we put the migration SQL in as a comment. I'm not sure if that's a practice we want to carry over to greenfield Rails projects.

Reply:
Usually, we don't run migrations in production, I'm willing to be wrong but I think this practice expands to all ruby/rails projects.

Reply:
@rvalenciano 🌴

Reply:
Again, I'm not wedded to anything here: happy to do what the gang thinks is best.